reactDashboard/src/components/Utils/Sonification.js

```diff
@@ -160,20 +180,40 @@ export function calculateMaxDataValue(seriesData = null) {
  * @param {number[]} seriesData - Dataset to sonify.
  * @param {string} [place = "Scotland"] - region name for dataset. If undefined, defaults to 'Scotland'.
```

Comment:
Missing a parameter in the docs here

Reply:
added: 

 * @param {object} dateRange - startDate and endDate for intro speech.